src/controller/artifact/controller.go

```diff
@@ -73,6 +74,7 @@ var (
 		chart.ArtifactTypeChart: icon.DigestOfIconChart,
 		cnab.ArtifactTypeCNAB:   icon.DigestOfIconCNAB,
 		wasm.ArtifactTypeWASM:   icon.DigestOfIconWASM,
+		sbom.ArtifactTypeSBOM: icon.DigestOfIconAccSBOM,
```

Comment:
gofmt? the `icon.DigestOfIconAccSBOM,` should be vertically align with the other values.